builder/csharp_builder.rb

```diff
@@ -53,13 +54,18 @@ def list_disposable_files
   end
 
   def create_project(mand_files, disp_files)
-    tmpl = File.read(@project_template)
-    files_xml = (mand_files + disp_files).map { |x| "    <Compile Include=\"#{x}\" />" }.join("\n")
-    project = tmpl.gsub(/%%%FILES%%%/, files_xml)
-    File.write(@project_file, project)
+    Dir[@compiled_dir + "/**/*.{cs}"].each do |original_parser_filepath| 
+      new_parser_filepath = original_parser_filepath.gsub(@compiled_dir, @spec_dir + "/parsers/")
```

Comment:
Ok, I see that, given new approach does not list files to compile inside csproj, it means that all files have to be compiled. To work with that paradigm, we will not be just creating project file, but rather maintain a copy of parsers generated in `compiled/csharp`. My questions to you then:

* Is it necessary / mandatory for this copy to be in `@spec_dir`?
* If it is, how do we manage life cycle of that directory?
  * Shall we add it to gitignore to avoid people adding its contents to the repo?
  * How do we ensure that every subsequent attempt of the build will not pick up files copied on previous attempt (which proved to be causing build problems)?
    * Shall we clear this completely first thing in `create_project` + recopy files?
    * Shall we copy all files initially and then delete files proven to be broken after build analysis?

Reply:
You are right. I have not realized that you may want to build it multiple times, removing the broken test+parser each iteration.

I will modify this PR.

> Is it necessary / mandatory for this copy to be in @spec_dir?
It can be linked from other directory in the same way how you had it.

Reply:
Now the `csproj` is committed in the repository and by default links all parsers, so it behaves as expected for local development. But, in `create_project` the files are listed explicitly as before.